HLTrigger/Configuration/python/HLT_75e33/modules/hltParticleFlowClusterHBHE_cfi.py

```diff
@@ -59,7 +59,7 @@
         ),
         clusterTimeResFromSeed = cms.bool(False),
         excludeOtherSeeds = cms.bool(True),
-        maxIterations = cms.uint32(50),
+        maxIterations = cms.uint32(5),
```

Comment:
As far as I understand, this changes the physics outputs of the Jet/MET HLT Phase-2 reco. Should I assume this change is okay with Jet/MET ?

Reply:
I think it would be good if they explicitly sign it off.

Reply:
@alpakpinar @theochatzis , I guess we wait for your feedback.

Reply:
Hello, we did some preliminary tests on performance and some thresholds for changing the iterations (you can see the results on the slides [here](https://indico.cern.ch/event/1265496/contributions/5314462/subcontributions/418047/attachments/2613316/4515832/preliminaryChecks.pdf) ). From these we see no big effect from this change.

Reply:
Thanks a lot Theo! Please @Martin-Grunewald and @missirol consider that you have the green light from HLT Upgrade!